bundles/org.openhab.binding.surepetcare/src/main/java/org/openhab/binding/surepetcare/internal/handler/SurePetcareBaseObjectHandler.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.surepetcare.internal.handler;
+
+import static org.eclipse.smarthome.core.thing.ThingStatus.ONLINE;
+
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.cache.ExpiringCache;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.surepetcare.internal.SurePetcareAPIHelper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link SurePetcareBaseObjectHandler} is responsible for handling the things created to represent Sure Petcare
+ * objects.
+ *
+ * @author Rene Scherer - Initial Contribution
+ */
+@NonNullByDefault
+public abstract class SurePetcareBaseObjectHandler extends BaseThingHandler {
+
+    private static final int UPDATE_THING_CACHE_TIMEOUT = 3000; // 3 secs
```

Comment:
You can also pass a duration to the expiring cache then you don't need the comment but can actually code it :-)

Reply:
done